### ISSUE TITLE
Create UI for Swap

### ISSUE THREAD

**Issue by harshita-srivastava-yral** [dolr-ai/hot-or-not-web-leptos-ssr#506]

**Comment by harshita-srivastava-yral** [comment]
- Swap changes done and added in a new repo
- This repo also has changes for Design UAT of PumpAI

**Comment by harshita-srivastava-yral** [comment]
- Updated the design to match what design suggested.

**Comment by harshita-srivastava-yral** [comment]
- Discussion to be done today on Swap UI and landing page of icpumpfun

**Comment by harshita-srivastava-yral** [comment]
- Swap UI has been updated and link will be shared today

**Comment by harshita-srivastava-yral** [comment]
- Pending on design to for UAT closure. ETA: Today

**Comment by harshita-srivastava-yral** [comment]
- UAT changes have been comepleted
- Design to verify the changes as next step

**Comment by siyara-m-yral** [comment]
- Backend integration pending and UAT will be needed again
- PR will change with the integration

**Comment by siyara-m-yral** [comment]
Is this implemented? @ravi-sawlani-yral

**Comment by saikatdas0790** [comment]
No, we're shelving this for now. New issue will be created and linked if this is prioritized again in the future
